frontend/app/module/polling_stations/page/PollingStationListPage.tsx

```diff
@@ -0,0 +1,55 @@
+import { PollingStationType, usePollingStationList } from "@kiesraad/api";
+import { IconChevronRight } from "@kiesraad/icon";
+
+function pollingStationType(pollingStationType: PollingStationType): string {
+  switch (pollingStationType) {
+    case "Bijzonder":
+      return "Bijzonder";
+    case "Mobiel":
+      return "Mobiel";
+    case "VasteLocatie":
+      return "Vaste locatie";
+  }
+}
+
+export function PollingStationListPage() {
+  const { pollingStations } = usePollingStationList();
+
+  return (
+    <>
+      <header>
+        <section>
+          <h1>Stembureaus</h1>
+        </section>
+      </header>
+      <main>
+        <article>
+          <table id="polling_stations" className="polling_stations_table">
```

Comment:
Suggestion for later: Create Table component with css module

Reply:
@lkleuver We have already created an issue for this: https://github.com/kiesraad/abacus/issues/433 :)